compose.yaml

```diff
@@ -0,0 +1,24 @@
+services:
+  odoo:
```

Comment:
Could you port the 8069 and 8072, please?

In order to be able to test the instance directly using URL:8069 or using nginx of the host